typescript/phoenix-sdk/src/utils/market.ts

```diff
@@ -197,9 +217,26 @@ export function getNodeIndices<Key, Value>(
   return indexMap;
 }
 
+/**
+ * Takes a raw order ID and returns a displayable order sequence number
+ *
+ * On the raw order book, sequence numbers are stored as unsigned 64-bit integers,
+ * with bits inverted for bids and left as-is for asks. This function converts
+ * the raw order ID to a signed 64-bit integer, and then converts it to a
+ * displayable order sequence number.
+ *
+ * @param orderId
+ */
+export function getUiOrderSequenceNumber(orderId: OrderId): BN {
```

Comment:
double-check this @jarry-xiao

Reply:
Let's write a few test cases for 32 bit integers to sanity check.

(2^32 - i).fromTwos(32) should yield -i for all i <= 2^31